src/io/flutter/samples/FlutterSample.java

```diff
@@ -42,7 +42,23 @@ public String toString() {
    * Get a label suitable for display in a chooser (e.g., combobox).
    */
   public String getDisplayLabel() {
-    // TODO(pq): add disambiguation once it's needed.
+    // TODO(pq): come up with disambiguated labels.
+    // TODO(pq): consider adding (package suffix too).
+    // Index isn't quite enough for disambiguation (there are still dups like DeletableChips).
+    //final String[] parts = id.split("\\.");
+    //final String lastPart = parts[parts.length-1];
+    //
+    //String suffix = "";
+    //try {
+    //  final int index = Integer.parseInt(lastPart);
+    //  if (index != 1) {
+    //    suffix = " (" + index + ")";
+    //  }
+    //} catch (NumberFormatException e) {
+    //  // ignore
+    //}
+    //
+    //return getElement() + suffix;
```

Comment:
I'm assuming you want to keep all the above in case it is useful solving the TODOs. If you just forgot about it, then here's a reminder to delete it. :)

Reply:
Intentional.  But thanks!